--- kadu-core/emoticons.cpp	2008-09-20 20:13:23.000000000 +0200
+++ kadu-core/emoticons.new.cpp	2009-12-29 11:01:43.652703023 +0100
@@ -354,7 +354,9 @@
 EmoticonSelectorButton::EmoticonSelectorButton(const QString& emoticon_string, const QString& anim_path, const QString& static_path, QWidget *parent)
 	: QToolButton(parent), EmoticonString(emoticon_string), AnimPath(anim_path), StaticPath(static_path), Movie(0)
 {
-	setPixmap(QPixmap(StaticPath));
+	QPixmap p(StaticPath);
+	setPixmap(p);
+	setIconSize(QSize(18, 18));
 	setAutoRaise(true);
 	setMouseTracking(true);
 	setToolTip(emoticon_string);
@@ -388,6 +390,7 @@
 	if (Movie == NULL)
 	{
 		Movie = new QMovie(AnimPath);
+		Movie->setScaledSize(QSize(18,18));
 		Movie->start();
 		connect(Movie, SIGNAL(updated(const QRect &)), this, SLOT(movieUpdate()));
 	}
