diff --git a/_paludis_wrapper.bash b/_paludis_wrapper.bash
index 04fef6f..9e19a72 100644
--- a/_paludis_wrapper.bash
+++ b/_paludis_wrapper.bash
@@ -69,6 +69,10 @@ done
 export PALUDIS_HOOKS_ASK_PPID=${PPID}
 export PALUDIS_HOOKS_ASK=${ask}
 
+if [[ `eselect paludis-hook list | grep generate-metadata | grep enabled` ]] ; then
+        export PALUDIS_NO_WRITE_CACHE_CLEAN=yes
+fi
+
 trap "exit " EXIT SIGINT SIGTERM
 
 if [ ${PALUDIS_WRAPPER_NICE} == "yes" ]; then
