--- niceview.pro.old	2009-05-03 22:46:10.000000000 +0200
+++ niceview.pro	2009-05-03 22:46:23.000000000 +0200
@@ -36,14 +36,10 @@
     INSTALL_PREFIX = /usr
     icons.path = /usr/share/icons
     icons.files = icons/$${TARGET}48.png
-    doc.path = /usr/share/doc/wiitcpload
-    doc.files = doc/CHANGELOG \
-        doc/COPYING
     desktop.path = /usr/share/applications
     desktop.files = res/$${TARGET}.desktop
     target.path = $${INSTALL_PREFIX}/bin
     INSTALLS = target \
-        doc \
         icons \
         desktop
 }
